### ISSUE TITLE
Create Actionable Steps Spreadsheet for NC Based on SR Type

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/311-data#1745]
### Overview
We will want to come up with a way to track actionable steps for NCs to deal with service requests of each type

### Action Items
- [ ] create spreadsheet "SR Actionable Steps for Neighborhood Councils" in PM folder
- [ ] add columns: SR Type, Description
- [ ] populate at least 2 rows (i.e. "Bulky Items","LA Waste Management provides XYZ services for free N times a year. Consider scheduling a regular pickup at address, M, every quarter")

**Comment by ryanfchase** [comment]
Requesting review of this ticket!
@bberhane @cottonchristopher

**Comment by bberhane** [comment]
@ryanfchase Just to clarify, does this pertain to the use cases Bonnie discussed with us in our PM meeting?

**Comment by ryanfchase** [comment]
@bberhane yes. While Bonnie didn't ask for a whole spreadsheet, I think it might be a good idea to track them as a list.

**Comment by bberhane** [comment]
Approved!

**Comment by bberhane** [comment]
Hi @cottonchristopher,

Please leave a comment with the following items:

- updated ETA
- progress from the last week (if applicable)
- availability for communications during the week

**Comment by cottonchristopher** [comment]
Use-Case Spreadsheet SR types:  https://docs.google.com/spreadsheets/d/1kFZ8OGlX0sHt0ZDc2vb9aexgJ0iiUVes15alOGL4_gk/edit?gid=0#gid=0
    - will pick this ticket back up soon

**Comment by ryanfchase** [comment]
@cottonchristopher Thanks for your work on this! This is a good start, let's review it with Bonnie at the PM meeting tmo. I'm going to move it to "In Review".

**Comment by ryanfchase** [comment]
I'm going to move this to Questions with ready for product, since we are actually waiting on input from Bonnie, and we have a meeting scheduled with her tomorrow

**Comment by ryanfchase** [comment]
Question for Bonnie:

We have assembled a few use-cases (aka actionable steps) for community members. They include:
- reminders of the available municipal services that a member should consider taking action with
- suggestions on what kind of information a member might need that is relevant to the SR
- suggestions on what types of data might supplement or explain the appearance of a particular SR type
- listing local organizations and institutions that may be interested in helping address the SR

Can you please review and let us know if we're on the right track?
